docs/plugins/development/plugin-descriptor-file.asciidoc

```diff
@@ -0,0 +1,37 @@
+["source","properties"]
+:plugin-properties-files: {elasticsearch-root}/build-tools/src/main/resources
+[source,yaml]
+----
+include::{plugin-properties-files}/plugin-descriptor.properties[]
+----
+
+Either fill in this template yourself or, if you are using Elasticsearch's Gradle build system, you
+can fill in the necessary values in the `build.gradle` file for your plugin.
+
+[discrete]
+==== Mandatory elements for plugins
```

Comment:
I wonder if instead of showing the descriptor embedded on the page we could just have a link to a separate page.
The preference would be that user would use a gradle plugin
Something along the lines
```
"use esplugin configuration to define properties necessary to create plugin descriptor.
the simplest configuration is:
esplugin {
  name 'stable-analysis-plugin'
  description 'An example analysis plugin using stable plugin api'
}
see more properties in the LINK or create the properties file manually if you are not using elasticsearch.stable-esplugin
```
and under LINK we would direct to the stable-plugin-descriptor.properties page
WDYT?

Reply:
That's a great idea. I have made that change.